From: Takashi Iwai <tiwai@suse.de>
Subject: kABI workaround for snd_hda_bus.bus_probing addition
Patch-mainline: No, SLE15 only
References: bsc#1122944

A newly added bit field has to be hidden for kABI compatibility.

Signed-off-by: Takashi Iwai <tiwai@suse.de>

---
 sound/pci/hda/hda_codec.h |    2 ++
 1 file changed, 2 insertions(+)

--- a/sound/pci/hda/hda_codec.h
+++ b/sound/pci/hda/hda_codec.h
@@ -68,7 +68,9 @@ struct hda_bus {
 	unsigned int response_reset:1;	/* controller was reset */
 	unsigned int in_reset:1;	/* during reset operation */
 	unsigned int no_response_fallback:1; /* don't fallback at RIRB error */
+#ifndef __GENKSYMS__
 	unsigned int bus_probing :1;	/* during probing process */
+#endif
 
 	int primary_dig_out_type;	/* primary digital out PCM type */
 	unsigned int mixer_assigned;	/* codec addr for mixer name */
